layers/transportation_name/update_transportation_name.sql

```diff
@@ -57,12 +55,8 @@ SELECT (ST_Dump(geometry)).geom                AS geometry,
 FROM (
          SELECT ST_LineMerge(ST_Collect(geometry)) AS geometry,
                 name,
-                name_en,
-                name_de,
-                hstore(string_agg(nullif(slice_language_tags(tags ||
-                                                             hstore(ARRAY ['name', name, 'name:en', name_en, 'name:de', name_de]))::text,
-                                         ''), ','))
-                                                   AS "tags",
+                hstore(string_agg(nullif(slice_language_tags(tags || hstore(ARRAY ['name', name]))::text, ''),
+                                  ',')) AS "tags",
```

Comment:
```suggestion
                hstore(string_agg(nullif(slice_language_tags(tags)::text, ''), ',')) AS "tags",
```

I can see why `name_en` and `name_de` were originally added to `tags` - their value **was** computed earlier using `name`, `name:en` and `name:de`:
https://github.com/openmaptiles/openmaptiles/blob/fb7c1ef42b4fdb21f98466274b8e4636047739cb/layers/transportation_name/layer.sql#L27-L28

I fail to see the purpose of adding `name` to `tags` - `name` is included in the result as a separate column and it is not used by the `{name_languages}` expression:
https://github.com/openmaptiles/openmaptiles/blob/9bcc51919d3f6167e92cd10f696e1cecfec81c0b/layers/transportation_name/transportation_name.yaml#L87